x-pack/plugins/transform/public/app/hooks/use_search_items/common.ts

```diff
@@ -6,68 +6,31 @@
  */
 
 import { buildEsQuery } from '@kbn/es-query';
-import { SavedObjectsClientContract, SimpleSavedObject, IUiSettingsClient } from '@kbn/core/public';
+import { IUiSettingsClient } from '@kbn/core/public';
 import { getEsQueryConfig } from '@kbn/data-plugin/public';
-import { DataView, DataViewAttributes, DataViewsContract } from '@kbn/data-views-plugin/public';
+import { DataView, DataViewsContract } from '@kbn/data-views-plugin/public';
```

Comment:
```suggestion
import type { DataView, DataViewsContract } from '@kbn/data-views-plugin/public';
```

Reply:
Done in https://github.com/elastic/kibana/pull/131819/commits/36d639885f4609acb2982b083b9639e7a47b7956